docs/850_roadmap/index.md

```diff
@@ -29,38 +29,59 @@ Uit onderzoek is gebleken dat 92% van de aanvragen standaardaanvragen zijn. Deze
 1. [Beëindigen borgstelling]
 1. [Schuldenknooppunt adapter]
 
+## MVP meerwerk
+
+Op basis van voortschrijdend inzicht zijn de volgende features nog noodzakelijk om live te kunnen gaan:
+
+1. De afwijsreden toevoegen aan de [taak](../producten/borgstelling/aanvragen-borgstelling/accepteren-maatwerk-aanvraag/index.html#beoordeel-aanvraag) en de [brief](../producten/borgstelling/aanvragen-borgstelling/index.html#archiveer-afwijzing).
+1. De portefeuille overname datum toevoegen aan [kredietbank beheer](../producten/kredietbanken-beheeren/index.html).
+1. Voor portefeuille overnames moet het uitstaand saldo gebruikt worden i.p.v. de bruto kredietsom bij het [registreren van de verkoop](../producten/borgstelling/aanvragen-borgstelling/index.html#registreer-verkoop).
+1. [Borgstellingen categorien](../producten/borgstelling-categorien-beheeren/index.html) realiseren en implementeren voor het vaststellen van de premiepercentage en de automatische acceptatie.
```

Comment:
@arjangeertsema Wat is de logica van wanneer welke borgstelling categorie gebruikt moet worden?

Reply:
Borgstelling categorie komt via de statistiek code. Wanneer er geen borgstelling categorie vastgesteld kan worden dan gaat het om een standaard aanvraag.